### PR TITLE
added option to remove the filesize restriction

### LINKED ISSUE #2131: Upload filesize needs to be 1e+89
999999999999999999999999999999999999999999999999999999999999999999999999999999
= 1e+89 

However, rocket chat thinks this number is equal to 1, so we can't upload anything.
We use huge files here at work, bigger than your average company.

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core 
It closes #2131
If max file size is set to -1 it will remove the size restriction.
@engelgabriel is this what you wanted?

### CODE REVIEW

**Comment by marceloschmidt** [comment]
@MartinSchoeler why not simply leave it blank?

**Comment by MartinSchoeler** [comment]
@marceloschmidt It was a @engelgabriel idea. Here https://github.com/RocketChat/Rocket.Chat/issues/2131#issuecomment-246902588

**Comment by marceloschmidt** [comment]
So he can answer my question.

**Comment by engelgabriel** [comment]
@marceloschmidt and @MartinSchoeler use the empty for no size restriction, but let's add something to the description like: "Leave it blank for no restriction"

File: packages/rocketchat-file-upload/lib/FileUpload.js [hunk 1]
```diff
@@ -18,11 +18,13 @@ FileUpload = {
 			return false;
 		}
 
-		if (file.size > maxFileSize) {
-			const reason = TAPi18n.__('File_exceeds_allowed_size_of_bytes', {
-				size: filesize(maxFileSize)
-			}, user.language);
-			throw new Meteor.Error('error-file-too-large', reason);
+		if (parseInt(maxFileSize) !== -1) {
```
Comment (engelgabriel) [inline packages/rocketchat-file-upload/lib/FileUpload.js]: Maybe just compare to > 0?

Comment (engelgabriel) [inline packages/rocketchat-file-upload/lib/FileUpload.js]: @RocketChat/core what do you think?

Comment (marceloschmidt) [inline packages/rocketchat-file-upload/lib/FileUpload.js]: Agreed!

**Comment by marceloschmidt** [comment]
I agree! That sounds like a better solution

**Pull request merged**